docs/changelog.rst

```diff
@@ -1,6 +1,26 @@
 Changelog
 =========
 
+0.3.0 (Unreleased)
+------------------
+
+Changed
+~~~~~~~
+
+- Refactor to build upon `ipython-sql <https://pypi.org/project/ipython-sql/>`__.
+  Removes several functions that can be replaced with direct usage of ipython-sql magics in the notebook,
+  and replace all remaining sql calls with calls to ipython-sql.
+
+  Replacements (must run ``%load_ext sql`` first, and ``%config SqlMagic.autopandas = True`` to get a pandas ``DataFrame``):
+
+  * :meth:`ocdskingfishercolab.create_connection` — replaced by using an ipython-sql connection string, e.g. ``%sql postgresql://user:pass@host/db``
```

Comment:
Use `-  ` instead of `* ` for consistency with other changelog entries.

Reply:
For all these removed functions, :meth: will refer to nothing and, I think, will cause a warning in the Sphinx build. We might as well just put double backticks.

Reply:
I don't see any warnings in the Sphinx build.

`:meth:` is formatted differently to double backticks (it's black):

![Screenshot from 2020-12-08 11-21-24](https://user-images.githubusercontent.com/634/101477749-8f071780-3947-11eb-8224-3fb4e1072208.png)

(But, we might prefer the method names that aren't links to look different to those with working links).

Reply:
Let's use double backticks to visually distinguish old methods.